.env.example

```diff
@@ -19,26 +47,65 @@ CHRONOS_SCHEDULE_JOBS='0 */10 * * * *'
 CHRONOS_UPDATE_TOKENS='0 0 0 1,15 * *'
 CHRONOS_PROCESS_RECURRENCE='0 */5 * * * *'
 CHRONOS_AUTOPAUSE='0 0 5 * * *'
-CI='true'
+
+# DATABASES
+POSTGRES_PASSWORD=parabol
+POSTGRES_USER=pgparaboladmin
+POSTGRES_DB=parabol-saas
+POSTGRES_HOST=localhost
+POSTGRES_PORT=5432
+# POSTGRES_POOL_SIZE=5
+# POSTGRES_SSL_REJECT_UNAUTHORIZED=false
+# POSTGRES_SSL_DIR='/var/lib/postgresql'
+REDIS_PASSWORD=''
+REDIS_URL='redis://localhost:6379'
+# REDIS_TLS_CERT_FILE=./certs/redis.crt
+# REDIS_TLS_KEY_FILE=./certs/redis.key
+# REDIS_TLS_CA_FILE=./certs/redisCA.crt
+# REDIS_TLS_REJECT_UNAUTHORIZED='false'
+RETHINKDB_URL='rethinkdb://localhost:28015/actionDevelopment'
+RETHINKDB_SSL='false'
+
+# DATADOG
 DD_AGENT_HOST='172.17.0.1'
-DD_API_KEY='DD_API_KEY'
 DD_APM_ENABLED='false'
+DD_API_KEY=''
+DD_APPLICATIONID=''
+DD_APPSEC_ENABLED='true'
+DD_CLIENTTOKEN=''
+DD_ENV=''
+DD_LOGS_INJECTION='true'
+DD_RUNTIME_METRICS_ENABLED='true'
+DD_SERVICE='DD_SERVICE_NAME'
+DD_TRACE_AGENT_URL='PROTOCOL://URL:PORT'
 DD_TRACE_ENABLED='false'
-DD_ENV='local'
-FILE_STORE_PROVIDER='s3'
+
+# GOOGLE
+GA_TRACKING_ID='' # GOOGLE ANALYTICS TRACKING ID
```

Comment:
-2 inline comments are invalid for .env files (at least for the parser that we use!)

`process.env.GA_TRACK_ID` should yield an empty string

i realize that we set a bad pattern here that was already in place with `MAIL_SMTP_USE_TLS`, but since that's never used it never triggered.

Reply:
Comment moved to upper line + line commented out